ethereum/contracts/state-transition/chain-deps/facets/Admin.sol

```diff
@@ -65,15 +66,15 @@ contract AdminFacet is Base, IAdmin {
     /// @param _validator Validator address
     /// @param _active Active flag
     function setValidator(address _validator, bool _active) external onlyGovernorOrAdmin {
-        s.validators[_validator] = _active;
+        chainStorage.validators[_validator] = _active;
         emit ValidatorStatusUpdate(_validator, _active);
     }
 
     /// @notice Change zk porter availability
     /// @param _zkPorterIsAvailable The availability of zk porter shard
     function setPorterAvailability(bool _zkPorterIsAvailable) external onlyGovernor {
```

Comment:
Nit: I would probably not give the governors of the chain the power to update this one yet. Changing it right now is a shot in the foot